polkadot-parachains/src/chain_spec.rs

```diff
@@ -34,6 +34,9 @@ pub type ShellChainSpec = sc_service::GenericChainSpec<shell_runtime::GenesisCon
 pub type SeedlingChainSpec =
 	sc_service::GenericChainSpec<seedling_runtime::GenesisConfig, Extensions>;
 
+/// The default XCM version to set in genesis config.
+const SAFE_XCM_VERSION: u32 = 2;
```

Comment:
This makes no real sense. The xcm crate already exposes the latest `XCM_VERSION` and that should be used here.

Reply:
It exposes the *latest* version, which means that the xcm version in the chain spec would change when updating Polkadot/XCM. I don't think that's what we want. I think we want to use the known-good version and then update conservatively/in our own time. (i.e. when v3 is published, we want to default to v2 for the safe fallback version until we decide to upgrade).

Reply:
Why? These are development chain specs and why do you not want to use the latest version there? Anything else doesn't make really sense, because for development you don't really care what is the latest "commonly" deployed version.

Reply:
Mmmh, I would've assumed on a fresh checkout you would want the chain specs to be fairly close to what's deployed.
Will impact any integration tests for example.

Reply:
Where do / should the prod chain specs live? I would like to think that the prod chain specs would still be stored in master and then the release branch have the smallest delta possible.

Reply:
the prod chain specs are in `polkadot-parachains/res`, so e.g. `polkadot-parachains/res/statemint.json`

Reply:
> Mmmh, I would've assumed on a fresh checkout you would want the chain specs to be fairly close to what's deployed. Will impact any integration tests for example. Edit: Could see this going either way shrug

How should that work? If you add some new pallet or whatever, the chain spec may changes. There is tons of reasons why they could differ. 

For tests you probably want to send explicit XCM versions anyway. So, having cross chain transfers using V2 and V3. Maybe later with V4. Etc.